src/routes/papers/index.js

```diff
@@ -24,12 +26,12 @@ function getBadgeCount(request, response) {
     return;
   }
 
-  var getBadges = badgerService.getBadges(null, null, {
```

Comment:
Hey @josmas, looking through this now.

Instead of renaming `getBadges`, it might be best `getBadgeCount` actually calls the [original `getBadges`](https://github.com/josmas/PaperBadger/blob/mofo_update/src/routes/papers/index.js#L11), but add an option to return the count instead of actual badges. Thoughts?

Let me know if you want to hop on skype

Thoughts?